dynamic-plugins/wrappers/red-hat-developer-hub-backstage-plugin-marketplace/package.json

```diff
@@ -0,0 +1,65 @@
+{
+    "name": "red-hat-developer-hub-backstage-plugin-marketplace",
```

Comment:
Can we use 2 spaces as intenion. This will will be updated later automatically and I like to have these update-PRs as small as possible.